workflow_scripts/config.py

```diff
@@ -0,0 +1,5 @@
+models_info = [
+    # (script_path, model_name, model_zoo_path)
+    ("torch_hub/alexnet.py", "alexnet_torch_hub_7df2a577", "alexnet/alexnet.onnx"),
+    ("torchvision/fasterrcnn_resnet50_fpn_v2.py", "fasterrcnn_resnet50_fpn_v2_torchvision_ec445cac", "fasterrcnn_resnet50_fpn_v2/fasterrcnn_resnet50_fpn_v2.onnx"),
```

Comment:
TODO: the commit ID here was changed recently... the script in this repo should find the right model without knowing the hard-coded ID to prevent failures.

Reply:
We dont need hardcoded paths to the model name. The hashes for the models will be consistent with the version of mlagility used.